stringer.go

```diff
@@ -138,16 +139,18 @@ type Generator struct {
 
 // Printf prints the string to the output
 func (g *Generator) Printf(format string, args ...interface{}) {
-	fmt.Fprintf(&g.buf, format, args...)
+	_, _ = fmt.Fprintf(&g.buf, format, args...)
 }
 
 // File holds a single parsed file and associated data.
 type File struct {
 	pkg  *Package  // Package to which this file belongs.
 	file *ast.File // Parsed AST.
 	// These fields are reset for each type being generated.
-	typeName string  // Name of the constant type.
-	values   []Value // Accumulator for constant values of that type.
+	typeName    string  // Name of the constant type.
+	values      []Value // Accumulator for constant values of that type.
+	trimPrefix  string
+	lineComment bool
```

Comment:
The two new fields added `trimPrefix` and `lineComment` doesn't seems to be used. Could you please remove it?